test/e2e/fixtures.go

```diff
@@ -237,17 +243,23 @@ func setupTestWithIPFIXCollector(tb testing.TB) (*TestData, bool, bool, error) {
 	}
 	ipfixCollectorAddr := fmt.Sprintf("%s:tcp", net.JoinHostPort(ipStr, ipfixCollectorPort))
 
-	tb.Logf("Applying flow aggregator YAML with ipfix collector address: %s", ipfixCollectorAddr)
-	if err := testData.deployFlowAggregator(ipfixCollectorAddr); err != nil {
+	tb.Logf("Deploying ClickHouse")
+	chPodIPs, err := testData.deployFlowVisibilityClickHouse()
```

Comment:
I just realized we are using the Pod IP to configure the Flow Aggregator. Do you know why we don't use the default instead (the Service DNS)?

Reply:
previously the dns name may have issue when tested together with other Antrea kind e2e test modes. reverted to the default svc dns name as the case is isolated now.